src/Tools/RapidResponseDesk/sections/AreaTime/AreaAndTimeSection.jsx

```diff
@@ -1,19 +1,234 @@
-import React from 'react';
+import React, { useEffect, useRef, useState } from 'react';
 import './AreaAndTimeSection.scss';
 import { connect } from 'react-redux';
 import CollapsiblePanel from '../../../../components/CollapsiblePanel/CollapsiblePanel';
 import { t } from 'ttag';
-import store, { collapsiblePanelSlice } from '../../../../store';
+import store, { areaAndTimeSectionSlice, collapsiblePanelSlice } from '../../../../store';
+import { AOISelection } from '../../../../components/AOISelection/AOISelection';
+import Slider from 'rc-slider';
+import { TimespanPicker } from '../../../../components/TimespanPicker/TimespanPicker';
+import moment from 'moment/moment';
+import { MIN_SEARCH_DATE } from '../../../../api/OData/ODataHelpers';
+import Button, { ButtonType } from '../../../../components/Button/Button';
 
 export const AreaAndTimeSectionProperties = Object.freeze({
   id: 'area-time',
   title: () => t`Area & Time`,
   toggleExpanded: (v) => store.dispatch(collapsiblePanelSlice.actions.setAreaTimeExpanded(v)),
 });
 
-const AreaAndTimeSection = ({ areaTimeExpanded }) => {
+const AreaAndTimeSection = ({
+  areaTimeExpanded,
+  aoiGeometry,
+  aoiIsDrawing,
+  mapBounds,
+  aoiCoverage,
+  minDate,
+  maxDate,
+}) => {
+  const [timespanArray, setTimespanArray] = useState([
+    {
+      from: moment.utc().subtract(1, 'month').startOf('day'),
+      to: moment.utc().endOf('day'),
+      id: 0,
+      displayCalendarFrom: false,
+      displayCalendarTo: false,
+    },
+  ]);
+
+  const minDateRange = moment.utc(minDate ? minDate : MIN_SEARCH_DATE).startOf('day');
+  const maxDateRange = moment.utc(maxDate).endOf('day');
```

Comment:
We should allow also dates in the future (I think that 1 year in advance should suffice) as it is possible to also place an order for future dates and view the results then when the time has come and it is delivered